containers/registry-protoc-gen-doc/README.md

```diff
@@ -0,0 +1,58 @@
+# Generating Documentation from protos
+
+We will be creating a custom controller which can generate html documentation
```

Comment:
Let's phrase this differently - I think "custom controller" suggests a change to the controller code. How about "We will be running the registry controller with a custom manifest which can generate..."